### DISCUSSION TITLE
Slack link is no longer active

### DISCUSSION THREAD

**Discussion by gustavo-leandro** [open-metadata/OpenMetadata#13774] [General]
I'm getting a "This link is no longer active. To join this workspace, you’ll need to ask the person who originally invited you for a new link." message when trying to join the slack workspace. 

Is the website link broken?

![Screenshot 2023-10-30 at 11 22 18](https://github.com/open-metadata/OpenMetadata/assets/83379623/d07fd450-b727-4277-a721-5abae1869509)

**Comment by troycurtisjr** [comment]
See also #13779
